lib/language.js

```diff
@@ -54,10 +54,14 @@ exports.errors = {
         length: 'must be {{limit}} bytes'
     },
     date: {
-        base: 'must be a number of milliseconds or valid date string',
+        base: 'must be a valid date string or timestamp',
```

Comment:
A timestamp might not always be in milliseconds but javascript expects that and so does a base joi date, otherwise it'd be a `timestamp.unix` error.